Modules/System Tests/Email/src/EmailMessageUnitTest.Codeunit.al

```diff
@@ -247,6 +248,54 @@ codeunit 134689 "Email Message Unit Test"
         Assert.ExpectedError(EmailMessageQueuedCannotInsertAttachmentErr);
     end;
 
+    [Test]
+    procedure SendMessageBccOnly()
+    var
+        TempEmailAccount: Record "Email Account" temporary;
+        EmailMessage: Codeunit "Email Message";
+        ConnectorMock: Codeunit "Connector Mock";
+        Recipients: List of [Text];
+        RecipientsCC: List of [Text];
+        RecipientsBCC: List of [Text];
+    begin
+        // Initialize
+        ConnectorMock.Initialize();
+        ConnectorMock.AddAccount(TempEmailAccount);
+
+        // [GIVEN] The message only has recipients in BCC
+        RecipientsBCC.Add('recipient@test.com');
+        EmailMessage.Create(Recipients, 'Test subject', 'Test body', true, RecipientsCC, RecipientsBCC);
+
+        // [WHEN] An email is sent
+        // [THEN] No error occurs
+        Email.Send(EmailMessage, TempEmailAccount."Account Id", TempEmailAccount.Connector);
+    end;
+
+    [Test]
+    procedure SendMessageNoRecipientsError()
+    var
+        TempEmailAccount: Record "Email Account" temporary;
+        EmailMessage: Codeunit "Email Message";
+        ConnectorMock: Codeunit "Connector Mock";
+        Recipients: List of [Text];
+        RecipientsCC: List of [Text];
+        RecipientsBCC: List of [Text];
+    begin
+        // Initialize
+        ConnectorMock.Initialize();
+        ConnectorMock.AddAccount(TempEmailAccount);
+
+        // [GIVEN] The message doesn't have any recipients
+        EmailMessage.Create(Recipients, 'Test subject', 'Test body', true, RecipientsCC, RecipientsBCC);
+
+        // [WHEN] An email is sent
+        // [THEN] A validation error occurs
+        Email.Send(EmailMessage, TempEmailAccount."Account Id", TempEmailAccount.Connector);
```

Comment:
Isn't an asserterror missing here?

Reply:
Yes, thanks for quickly catching the problem! I would expect the build workflow to show the failed test as well, will try to make some changes to make that happen